src/class.rs

```diff
@@ -53,7 +53,7 @@ pub struct Class {
     pub is_abstract: bool,
     pub internal: bool,
     pub internal_resolved: bool,
-    pub primary_ctor: bool,
+    pub has_constructor: bool,
 
     pub ctors: Vec<FctId>,
```

Comment:
I think this should now also be `Option<FctId>` instead of `Vec<FctId>`

Reply:
This should also make some code in typeck.rs unnecessary.

Reply:
Fixed. Thanks!